crates/turbo-tasks-malloc/src/lib.rs

```diff
@@ -4,6 +4,33 @@ use std::alloc::{GlobalAlloc, Layout};
 
 use self::counter::{add, flush, get, remove};
 
+#[derive(Default, Clone)]
+pub struct AllocationInfo {
+    pub allocations: usize,
+    pub deallocations: usize,
+    pub allocation_count: usize,
+    pub deallocation_count: usize,
+}
```

Comment:
Because this refers to thread-local data, and then does arithmetic with it assuming that it's still on the same thread, this struct should be explicitly `!Send`.

```suggestion
#[derive(Default, Clone)]
pub struct AllocationInfo {
    pub allocations: usize,
    pub deallocations: usize,
    pub allocation_count: usize,
    pub deallocation_count: usize,
    _not_send: PhantomData<dyn NotSend>,
}

trait NotSend: Sync {}
```

This is going to cause problems when actually trying to use this inside futures that need to be `Send` though, like in `CaptureFuture`.

Reply:
It's not "stored" in CaptureFuture, so it's fine